libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/atomic/molecules/ComposerAlertMolecule.kt

```diff
@@ -77,16 +97,28 @@ fun ComposerAlertMolecule(
                             avatarData = avatar,
                             avatarType = AvatarType.User,
                         )
+                    } else if (showIcon) {
+                        val icon = when (level) {
+                            ComposerAlertLevel.Default -> CompoundIcons.Info()
+                            ComposerAlertLevel.Info -> CompoundIcons.Info()
+                            ComposerAlertLevel.Critical -> CompoundIcons.Error()
+                        }
+                        val iconTint = when (level) {
+                            ComposerAlertLevel.Default -> ElementTheme.colors.iconPrimary
+                            ComposerAlertLevel.Info -> ElementTheme.colors.iconInfoPrimary
+                            ComposerAlertLevel.Critical -> ElementTheme.colors.iconCriticalPrimary
+                        }
+                        Icon(
```

Comment:
We do not use this version of `fun Icon` (we should maybe make it private). You can use
```
    Icon(
        imageVector = icon,
        tint = iconTint,
        contentDescription = null,
    )
```